starlite/config/compression.py

```diff
@@ -23,10 +23,9 @@ class CompressionConfig(BaseModel):
     'compression_config' key.
     """
 
-    backend: Union[CompressionBackend]
+    backend: Union[CompressionBackend, Literal["gzip", "brotli"]]
```

Comment:
Why not drop the enum entirely?

Reply:
I agree, I don't know that there is much value in keeping both.  Since using the Literal save an import, I think it makes sense to drop the enum

Reply:
cool, lets do that.